packages/core/stories/text-field.stories.js

```diff
@@ -320,49 +320,118 @@ storiesOf('Text Field', module)
+                type="text"
+                className="ray-text-field__input"
+                id="input"
+                placeholder="Few people are aware..."
+              />
+              <label className="ray-text-field__label" htmlFor="input">
+                Fun fact about Ray Eames
+              </label>
+            </div>
           </div>
         </div>{' '}
         <div className="ray-form-item">
           <div className="ray-text-field ray-text-field--disabled ray-text-field--with-icon-left">
-            <svg
-              className="ray-text-field__icon--left"
-              xmlns="http://www.w3.org/2000/svg"
-              viewBox="0 0 25 25"
-            >
-              <g id="budicon-profile-picture">
-                <path d="M12.5,4A4.5,4.5,0,1,0,17,8.5,4.5,4.5,0,0,0,12.5,4Zm0,8A3.5,3.5,0,1,1,16,8.5,3.504,3.504,0,0,1,12.5,12Zm0-12A12.4886,12.4886,0,0,0,5.0007,22.4834v0a12.4325,12.4325,0,0,0,14.9983,0v0q.5-.3761.9593-.7988l0,0A12.4869,12.4869,0,0,0,12.5,0Zm0,24a11.4432,11.4432,0,0,1-7.3931-2.7041,7.4887,7.4887,0,0,1,14.7863,0A11.4432,11.4432,0,0,1,12.5,24Zm8.25-3.5061a8.4871,8.4871,0,0,0-16.5,0,11.5,11.5,0,1,1,16.5,0Z" />
-              </g>
-            </svg>
+            <div className="ray-text-field__icon--left">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 25 25"
+                style={{ height: '1rem' }}
+              >
+                <g id="budicon-profile-picture">
+                  <path d="M12.5,4A4.5,4.5,0,1,0,17,8.5,4.5,4.5,0,0,0,12.5,4Zm0,8A3.5,3.5,0,1,1,16,8.5,3.504,3.504,0,0,1,12.5,12Zm0-12A12.4886,12.4886,0,0,0,5.0007,22.4834v0a12.4325,12.4325,0,0,0,14.9983,0v0q.5-.3761.9593-.7988l0,0A12.4869,12.4869,0,0,0,12.5,0Zm0,24a11.4432,11.4432,0,0,1-7.3931-2.7041,7.4887,7.4887,0,0,1,14.7863,0A11.4432,11.4432,0,0,1,12.5,24Zm8.25-3.5061a8.4871,8.4871,0,0,0-16.5,0,11.5,11.5,0,1,1,16.5,0Z" />
+                </g>
+              </svg>
+            </div>
+            <div className="ray-text-field__wrapper">
+              <input
+                type="text"
+                className="ray-text-field__input"
+                id="input2"
+                placeholder="Few people are aware..."
+                disabled
+              />
+              <label className="ray-text-field__label" htmlFor="input2">
+                Fun fact about Ray Eames
+              </label>
+            </div>
+          </div>
+        </div>
+      </>
+    );
+  })
+  .add('Text field, with prepend', () => {
+    setTimeout(initTextField);
 
-            <input
-              type="text"
-              className="ray-text-field__input"
-              id="input2"
-              placeholder="Few people are aware..."
-              disabled
-            />
-            <label className="ray-text-field__label" htmlFor="input2">
-              Fun fact about Ray Eames
-            </label>
+    return (
+      <>
+        <div className="ray-form-item">
```

Comment:
what is the difference between these two form items? possible copy/paste oop?
![Screen Shot 2019-05-09 at 3 10 57 PM](https://user-images.githubusercontent.com/19141291/57480113-dbbad600-726c-11e9-8608-ede8535a785a.png)

Reply:
Think the second one is intended for a disabled version (like text field with icon above it)

Reply:
yeah copy/paste job. ill make the 2nd one look different, or just remove it

Reply:
can use it for disabled state